guide/book.toml

```
@@ -25,3 +25,6 @@ boost-hierarchy = 2
 boost-paragraph = 1
 expand = true
 heading-split-level = 2
+
+[output.html.redirect]
+"/format/config.html" = "/format/configuration/index.html"
```

I don't think this will work, the path needs to be relative.  I think it needs to be something like this:

```suggestion
"/format/config.html" = "configuration/index.html"
```